include/aws/common/atomics.h

```diff
@@ -8,6 +8,8 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
+AWS_EXTERN_C_BEGIN
```

Comment:
Why are we exporting above the functions?

Reply:
resolving this. we're having this discussion on another comment...